plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -2228,7 +2228,10 @@ private long commitInsertOperation(
         long currentVersion = getMandatoryCurrentVersion(fileSystem, handle.location(), readVersion.get());
 
```

Comment:
Does this commit change logic in any way, or just a refactoring ?

Reply:
Just refactoring